packages/react-admin/src/mui/layout/Layout.js

```diff
@@ -85,7 +84,7 @@ const Layout = ({
         >
             <div className={classes.appFrame}>
                 <Hidden xsDown>
-                    <AppBar title={title} open={open} />
+                    <AppBar title={title} open={open} logout={logout} />
```

Comment:
This is a BC break for people with Custom layout. To be added to the UPGRADE guide, and please update the Theming (or Admin?) section about custom layouts.

Reply:
Added